### PR TITLE
modified TaskModuleMethods.java

### PR DESCRIPTION
**Pull request by gopal2602**
The framework modified
1. Added new method to TaskModuleMethods.java

### CODE REVIEW

**Comment by krish1002** [comment]
hello

File: src/test/java/methods/TaskModuleMethods.java [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
+	 */
+	public boolean loginToApp(WebDriver oDriver, String strUN, String strPWD)
+	{
+		String strStatus = null;
+		try {
+			test = extent.startTest("login To App");
+			strStatus+=appInd.setObject(oDriver, obj_UserName_Edit, strUN);
+			strStatus+=appInd.setObject(oDriver, obj_Password_Edit, strPWD);
+			strStatus+=appInd.clickObject(oDriver, obj_Login_Btn);
+			Thread.sleep(4000);
```
Comment (gopal2602) [inline src/test/java/methods/TaskModuleMethods.java:26]: Please remove this and call the reusable method

File: src/test/java/methods/TaskModuleMethods.java [hunk 2]
```diff
@@ -1,7 +1,57 @@
...
+				strStatus+=String.valueOf(appInd.clickObject(oDriver, obj_Explore_Btn));
+			}
+			Thread.sleep(4000);
+			
+			
+			//Handle the shortcut window
+			if(appInd.verifyOptionalElement(oDriver, obj_ShortCut_Window))
+			{
+				strStatus+=appInd.clickObject(oDriver, obj_shortCut_Close_Btn);
+			}
```
Comment (gopal2602) [inline src/test/java/methods/TaskModuleMethods.java:39]: Add s report here with screenshot

File: src/test/java/methods/TaskModuleMethods.java [hunk 3]
```diff
@@ -1,7 +1,57 @@
...
+			strStatus+=appInd.setObject(oDriver, obj_Password_Edit, strPWD);
+			strStatus+=appInd.clickObject(oDriver, obj_Login_Btn);
+			Thread.sleep(4000);
+			
+			//Verify optional element 
+			if(appInd.verifyOptionalElement(oDriver, obj_Explore_Btn)) {
+				strStatus+=String.valueOf(appInd.clickObject(oDriver, obj_Explore_Btn));
+			}
+			Thread.sleep(4000);
+
```
Comment (gopal2602) [inline src/test/java/methods/TaskModuleMethods.java:33]: remove hard coded waits and replace with webdriver waits

File: src/test/java/methods/TaskModuleMethods.java (hunk 1 above)
Comment (krish1002) [reply src/test/java/methods/TaskModuleMethods.java:26]: hello

**Comment by gopal2602** [comment]
REviewd and approved the new changes

**Pull request merged**